### ISSUE TITLE
Provide ability to retrieve OpenShift URL in machine readable format.

### ISSUE THREAD

**Issue by praveenkumar** [minishift/minishift#297]
Minishift offers currently offers:

```
minishift console --url
```

This, however, prints just the URL of OpenShift to the console. This is not in the export variable format as it used to be in CDK 2.x. For CDK-3.x we need to extend the console command to provide something like:

```
$  minishift console --machine-readable
HOST=192.168.99.110
PORT=8443
CONSOLE_URL=https://192.168.99.110:8443
```

**Comment by hferentschik** [comment]
See also https://issues.jboss.org/browse/CDK-47

**Comment by hferentschik** [comment]
Merged via pull request #310
